.github/workflows/aks_deploy_review.yml

```diff
@@ -0,0 +1,58 @@
+name: "Deploy review"
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Deploy environment"
+        required: true
+        default: review_aks
+        type: environment
+        options:
+          - review_aks
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - labeled
```

Comment:
wondering if this will always trigger a deploy vs this is triggered via adding a label. I think maybe we start with a label until we add the destroy workflow?

Reply:
Temporarily removed the `if` statement so we always build them, think we should add it back as a nice to have once we've seen this run a few times and are confident it's fully working